### ISSUE TITLE
Gourd and Mushrooms farms are missing inputs

### ISSUE THREAD

**Issue by ALEILGOD** [thedarkcolour/ForestryCE#141]
Gourd farm:
I can't place dirt in the gourd farm input slot, so the farm doesn't work(i already tried in the multifarm)

Mushroom farm:
i cant't place mushrooms in the mushroom farm to make it work(same thing)

I checked with what's that slot and it showed nothing

**Comment by thedarkcolour** [comment]
Yep, seems to be a regression. I messed up the farms during porting and never checked these.

**Comment by azure-hu** [comment]
Could the fix be backported to 1.19.2, or it is impossible?
